### PR TITLE
added example of cleaner configuration for k8s

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
I needed example of cleaner configuration for k8s. Was not able to find one on the web. So adding it here.

### CODE REVIEW

File: plugin/storage/es/README.md [hunk 1]
```diff
@@ -64,3 +64,26 @@ This script requires Docker to be running.
 Integration test framework for storage lie under `../integration`. 
 Add to `../integration/fixtures/traces/*.json` and `../integration/fixtures/queries.json` to add more
 trace cases.
+
+## Example of cleaner job configuration for Kubernetes
```
Comment (pavolloffay) [inline plugin/storage/es/README.md]: We don't want to document it here. There is an operator which handles k8s deployments.

The kubernetes repository contains a similar command for spark dependencies. The repository is deprecated but perhaps we could add a command for index cleaner there.

https://github.com/jaegertracing/jaeger-kubernetes#service-dependencies

```
kubectl run jaeger-spark-dependencies --schedule="55 23 * * *" --env="STORAGE=elasticsearch" --env="ES_NODES=elasticsearch:9200" --env="ES_USERNAME=changeme" --env="ES_PASSWORD=changeme" --restart=Never --image=jaegertracing/spark-dependencies
```

Comment (jpkrohling) [reply plugin/storage/es/README.md]: I think we could have an example here for reference, and add a notice saying that the recommended way to deploy Jaeger in Kubernetes is via the Operator.

Comment (pavolloffay) [reply plugin/storage/es/README.md]: There are already 3 repositories with kubernetes related docs and manifest (jaeger-kubernetes, jaeger-openshift, jaeger-operator). We do not have any kubernetes docs in this main repo.

My preference would be not adding it here.

Comment (dzmitry-lahoda) [reply plugin/storage/es/README.md]: so i may pull into https://github.com/jaegertracing/jaeger-kubernetes ? i am on minikube and aks, not openshift. we have tried jaeger-operator and offical es operator - these are too intrusive and do not fit.

Comment (dzmitry-lahoda) [reply plugin/storage/es/README.md]: so modified page of cleaner in jaegertracing/jaeger could point to page about cleaner at jaegertracing/jaeger-kubernetes or at jaeger-openshift, which could suggest to use jaeger-operator .

Comment (jpkrohling) [reply plugin/storage/es/README.md]: I would love to hear more about your experience with the operator. What made it a deal breaker?

**Comment by objectiser** [comment]
> we have tried jaeger-operator and offical es operator - these are too intrusive and do not fit (or i failed them to fit)

@dzmitry-lahoda it would be great if you could create an issue in the jaegertracing/jaeger-operator repo to outline your difficulties in using the operator to help us improve it.

File: plugin/storage/es/README.md (hunk 1 above)
Comment (pavolloffay) [reply plugin/storage/es/README.md]: https://github.com/jaegertracing/jaeger-kubernetes seems like a better place to document it. There is already documentation for spark dependencies job. Feel free to submit a PR there.


+1 on hearing why you cannot use the operator. We are constantly improving it and feedback really helps us to drive better adoption of the project.

**Comment by dzmitry-lahoda** [comment]
> 
> 
> > we have tried jaeger-operator and offical es operator - these are too intrusive and do not fit (or i failed them to fit)
> 
> @dzmitry-lahoda it would be great if you could create an issue in the jaegertracing/jaeger-operator repo to outline your difficulties in using the operator to help us improve it.

My story was:
```
I am new to k8s. I have deployed ElasticSearch as statefull set with Istio(need to disable Istio on cluster to cluster transport, found several issues in Istio backlog). Than I have tried ElasticSearch operator, and it failed with Istio (conflict on TLS, not sure if can put proper annotation to disable Istio for cluster to cluster transport). We had working Jaeger, but no operator. I have started to read page about Jaeger and found it too complex and potentially intrusive (potentially violating custom namespacing rules). So I have stayed on safe side (considering my time limitations either).
```
It would be simply for me to have `scenario - apply` set for each typical deployment, starting from most simple to less complex.  So operator per se may be fine, just me lacking proper experience to understand end to end.

**Pull request closed without merging**